bedrock/security/management/commands/update_security_advisories.py

```diff
@@ -147,13 +147,14 @@ def add_or_update_cve(data):
         except MitreCVE.DoesNotExist:
             cve = MitreCVE(**cve_data)
             cve.products = data['fixed_in']
+            cve.mfsa_ids.append(data['mfsa_id'])
         else:
-            cve.products = list(set(data['fixed_in']) | set(cve.products))
+            cve.products = list(set(cve.products).union(data['fixed_in']))
+            cve.mfsa_ids = list(set(cve.mfsa_ids).union([data['mfsa_id']]))
```

Comment:
Just curious - is `union` preferred over `|`?

Reply:
https://docs.python.org/2/library/stdtypes.html#frozenset.union